### DISCUSSION TITLE
NV14 Lua Scripts under Tools Menu

### DISCUSSION THREAD

**Discussion by Neukom720** [EdgeTX/edgetx#2103] [Q&A]
Hey Guys, I'm currently on 2.71 on my NV14 and on my sd card there wasn't a TOOLS folder so I added one and I added my ELRS v1Lua , ELRS v2 Lua, Crossfire Lua, and TBS Agent Lite Lua and none of them show up in the quick access tools menu. Any help would be greatly appreciated! Thanks Guys! Want to also say a big thanks to all the edgetx Dev's for making the NV14 supported and all the hard work you have done!!!

**Comment by Neukom720** [comment]
Anyone?

**Reply by pfeerick** [reply]
The correct path is `SCRIPTS\TOOLS` for it to show up in the Radio Settings / Tools menu... so double check that. Otherwise I have no immediate idea as I haven't had any issues with them not showing up. 

![image](https://user-images.githubusercontent.com/5500713/176800663-952d3ea1-f2ef-48bf-b7a9-51c224719663.png)

**Reply by Neukom720** [reply]
@pfeerick thanks man I'll try that tonight!!!!
